### ISSUE TITLE
Click on criteria name to take to its evaluation section on correct tab page (from criteria bar and summary page)

### ISSUE THREAD

**Issue by selinad** [ClinGen/clincoded#1064]
Curators would like to be able to click on a criterion from 

1. Criteria bar
- Column in Summary table where criterion listed

**Comment by selinad** [comment]
Another request for this came in today -- we should bump it up in priority, maybe for R19?

**Comment by wrightmw** [comment]
@selinad This is one of the most common requests that I have heard users ask for.

**Comment by jimmyzhen** [comment]
Hi @selinad, @wrightmw,

I've updated the instance with the changes we discussed during the demo last Friday (e.g. clicking a criteria takes the user to the panel in which the evaluation is located).

Please review and let me know if you have any questions:
https://jz-criteria-bar.demo.clinicalgenome.org/

**Comment by selinad** [comment]
@jimmyzhen This works perfectly!!!  I checked every criterion and it took me to the correct place.  Nicely done!

One small thing: I was tempted to hit the back button to get back to the top after hitting on a criterion in the bar.  Is it possible to control the back arrow in this case?  Right now it takes me out of the interpretation

@wrightmw Do you think we should have some text or some other way of altering curators to the fact the criteria are clickable?

**Comment by jimmyzhen** [comment]
@selinad,

Can you elaborate on how you wish to control the browser's back/history button behavior in this case?

**Comment by selinad** [comment]
@jimmyzhen anytime you hit back when you are in Interpretation, it takes you out of the Interpretation.  My memory is you don't have much control over this per browser.  It was more noticeable in this case as maybe the curator decides that's not where they will want to be and then hits back button and they are out.

**Comment by jimmyzhen** [comment]
@selinad, your scenario makes sense if the criteria in question is the 1st (that the curator clicks on). But if the criteria in question is the 3rd or 4th being clicked, using the browser's history/back button would only take the curator back to the previously viewed evaluation section.

**Comment by wrightmw** [comment]
@jimmyzhen This works great!
@selinad Were you thinking of a temporary message during R19 that would disappear with the next release, or a permanent message?

**Comment by selinad** [comment]
@jimmyzhen I see -- that's what I did, I guess.  In that case, I think we are good.  Thanks1

@wrightmw I was thinking of a permanent message - I suppose we could put it in the text upon rollover for each criterion?  "Click to go to this criterion's section"

**Comment by selinad** [comment]
@wrightmw good point re mouseover strings.  I think just some text below the bar?  
"Mouse over a criterion code to see its description; click on it to go to its evaluation section"

**Comment by jimmyzhen** [comment]
Hi @selinad, @wrightmw,

How does this look?
![image](https://user-images.githubusercontent.com/6956310/39846892-18d7eb74-53b3-11e8-9414-492cf86f3e2e.png)

**Comment by selinad** [comment]
Ditto to Fan dabby dozy!!

**Comment by selinad** [comment]
@jimmyzhen PM1 works like this:
![image](https://user-images.githubusercontent.com/11320314/40271689-8820698a-5b55-11e8-8746-8572a178e869.png)

But PM2 works like this:

![image](https://user-images.githubusercontent.com/11320314/40271693-a9731628-5b55-11e8-83ce-ad2f0ee0dc3b.png)

PS1 goes here the first time I click on it and then goes to right place if I go up and click again:
![image](https://user-images.githubusercontent.com/11320314/40271731-3979742e-5b56-11e8-8d5f-06871316e0d8.png)

All others look great.  Strange as I tested them all before....will wait for @wrightmw to check

**Comment by jimmyzhen** [comment]
@selinad,

I was able to reproduce the reported behavior (re: **PS1**) when I had the Chrome Devtools opened at the bottom of the browser window on my Macbook Pro. But when I had the Chrome Devtools opened on the (right) side of the window, or not having the Chrome Devtools opened at all, clicking **PS1** incurred the expected behavior.

This is the first time I have come across this quirky behavior, and I will need to do some research on the browser's DOM Web API.

Can you confirm whether you had the Chrome Devtools opened at the time you experienced the reported behavior?

**Comment by selinad** [comment]
Thx @jimmyzhen. I didn’t have dev tools open but it worked the second try each time if there’s a clue in that.  I.e when I went back up and tried again it worked

**Comment by selinad** [comment]
@jimmyzhen thx for trying to debug.  It's strange it works the second time.  Sounds like there's not much we can do about it now.

**Comment by selinad** [comment]
@jimmyzhen I tried again in Chrome for a different variant and it worked the first time.  Then I tried in Safari and it didn't work the first time, but it did work the 2nd.  It's almost like the first time you do it in the instance for a browser?

@wrightmw if you've tried, did you have trouble?

**Comment by wrightmw** [comment]
I didn't have the same issue until I had the Devtools open. I think this is not a showstopper because if you click a second time it works.

**Comment by selinad** [comment]
Strange!  Mine had nothing to do with dev tools. I agree it’s not worth further time trying to sleuth it out!

**Comment by jimmyzhen** [comment]
It has to do with the height of the browser window relative to the height of the screen. Having the Devtools opened at the bottom of the browser window is essentially reducing the height of the browser's viewport.

Also, the issue occurs only when jumping from a different tab (e.g. Population,  Basic Info) to the **MIssense** subtab on the **Variant Type** tab. Jumping to the **LOF** subtab wouldn't have been an issue, nor when jumping between the subtabs on the **Variant Type** tab.

Putting all these strangeness together, we've got a head-scratching behavior.

**Comment by selinad** [comment]
😱 Let’s refrain from going bald for the moment...!

**Comment by jimmyzhen** [comment]
Shall we move this ticket to the **Release Ready** column then?

**Comment by selinad** [comment]
yes!  thanks @jimmyzhen
